src/app/api/refresh/route.ts

```diff
@@ -0,0 +1,74 @@
+import { NextRequest } from 'next/server'
+
+type Client = {
+	id: string
+	res: any
+}
+
+const clients: Client[] = []
+
+function addClient(id: string, res: any): void {
+	clients.push({ id, res })
+}
+
+function removeClient(id: string): void {
+	const index = clients.findIndex((client) => client.id === id)
```

Comment:
should we add a lil check for clients have elements here?

Reply:
Huh, what do you mean?